api/v1alpha1/application_types.go

```diff
@@ -68,6 +68,12 @@ type ApplicationSpec struct {
 	//+kubebuilder:validation:Optional
 	Ingresses []string `json:"ingresses,omitempty"`
 
+	// If the application is external, and uses statkart.no or kartverket.cloud name, set this to false. This will 
+	// make the application reachable only from the internet. This is to override the default behaviour of 
+	// setting kartverket.cloud or statkart.no applications as internal.
+	//+kubebuilder:validation:Optional
```

Comment:
```suggestion
	//+kubebuilder:validation:Optional
       //+kubebuilder:default:=true
```
and remove `GetIsInternal` function